rpcd/playbooks/roles/rpc_maas/tasks/process.yml

```diff
@@ -0,0 +1,56 @@
+---
+# Copyright 2016, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# NOTE:
+# We are not using ensure_local_checks.yml here, because we need access to other
+# variables in a programatic way, which the combination of task and template can
+# not currently handle.
+#
+# Instead, we are defining a variable which contains the list of process names
+# to check for. This list is contained in the dicts in process_checks_list. This
+# is more simple than cobbling together hacks to accomplish the same outcome.
+
+#- set_fact:
```

Comment:
Is this commented section a previous version or supposed to be an example for the preceding comment explanation?